easybuild/easyblocks/generic/packedbinary.py

```diff
@@ -45,18 +45,18 @@ def extract_step(self):
         EasyBlock.extract_step(self)
 
     def install_step(self):
-        """Copy all unpacked source directories to install directory, one-by-one."""
-        try:
-            os.chdir(self.builddir)
-            for src in os.listdir(self.builddir):
-                srcpath = os.path.join(self.builddir, src)
-                if os.path.isdir(srcpath):
-                    # copy files to install dir via Binary
+        """Copy all unpacked source files/directories to install directory, one-by-one."""
+        os.chdir(self.builddir)
+        for src in os.listdir(self.builddir):
+            srcpath = os.path.join(self.builddir, src)
+            # we only handle the case of a single file and no install_cmd here
+            if os.path.isfile(src) and self.cfg.get('install_cmd', None) is None:
```

Comment:
You want to use full path here, no? So use `os.path.isfile(srcpath)`?

Reply:
Done